### PR TITLE
chore: Up sprinboot to version 3.2.2 and jetty to version 12.0.5

### PR DESCRIPTION
**Pull request by ovidijusnortal**
Closes: XRDDEV-2584, XRDDEV-2585

### CODE REVIEW

File: src/common/common-util/src/main/java/ee/ria/xroad/common/conf/globalconf/ConfigurationFile.java [hunk 1]
```diff
@@ -136,8 +136,8 @@ private static ContentIdentifier getContentIdentifier(String value) {
 
         Map<String, String> p = new HashMap<>();
 
-        String id = HttpField.valueParameters(value, p);
-        String instance = p.get(PARAM_INSTANCE);
+        String id = HttpField.getValueParameters(value, p);
+        String instance = StringUtils.unwrap(p.get(PARAM_INSTANCE), '\'');
```
Comment (ricardas-buc) [inline src/common/common-util/src/main/java/ee/ria/xroad/common/conf/globalconf/ConfigurationFile.java:140]: how the params have changed that now we need extra unwraping?

Comment (ovidijusnortal) [reply src/common/common-util/src/main/java/ee/ria/xroad/common/conf/globalconf/ConfigurationFile.java:140]: Not params, the utility for extracting them changed so it leaves for some reason those quotes

File: src/proxy/runproxytest [hunk 2]
```diff
@@ -18,6 +18,7 @@ then
 fi
 
 $javabin -Xmx2g\
+    -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005\
```
Comment (ricardas-buc) [inline src/proxy/runproxytest:21]: should it suspend by default? maybe leave at n

File: src/proxy/src/test/java/ee/ria/xroad/proxy/testsuite/MessageTestCase.java [hunk 3]
```diff
@@ -189,14 +189,14 @@ public String getDisabledNotice(ServiceId service) {
     /**
      * @return the server proxy HTTP handler
      */
-    public AbstractHandler getServerProxyHandler() {
+    public  Handler.Abstract getServerProxyHandler() {
```
Comment (ricardas-buc) [inline src/proxy/src/test/java/ee/ria/xroad/proxy/testsuite/MessageTestCase.java:192]: I have noticed this extra space in multiple places, maybe reformat bunch of files again?

**Review by ricardas-buc — CHANGES_REQUESTED** [review]
Overall ok, just few leftover things from development that should be fixes

**Review by ricardas-buc — APPROVED** [review]

**Pull request merged**